### PR TITLE
Fix for multiple migration heads.

### PR DESCRIPTION
**Pull request by KaleabTessera**
Fix for multiple migration heads on dev.

### CODE REVIEW

File: api/migrations/versions/8ba06d0a54ee_.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""empty message
+
+Revision ID: 8ba06d0a54ee
+Revises: 0023aeda53b0
+Create Date: 2020-02-14 07:13:05.049933
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '8ba06d0a54ee'
```
Comment (amritpurshotam) [inline api/migrations/versions/8ba06d0a54ee_.py]: You can probably delete this file

File: api/migrations/versions/d80784e674a8_.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Revision ID: d80784e674a8
+Revises: ('72548fc390d4', 'f8ab033698a2', '8ba06d0a54ee')
+Create Date: 2020-02-17 12:35:15.131604
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'd80784e674a8'
+down_revision = ('72548fc390d4', 'f8ab033698a2', '8ba06d0a54ee')
```
Comment (amritpurshotam) [inline api/migrations/versions/d80784e674a8_.py]: And then backspace `8ba06d0a54ee`

**Comment by KaleabTessera** [comment]
Hmm let me investigate this tomorrow. Dev was down and on circle ci it was the multiple head issue. Does a PR run a migration before getting approved ? I def want to investigate this :0

**Comment by amritpurshotam** [comment]
@KaleabTessera Well I think you already fixed it. The tests on Circle CI do a db upgrade, downgrade, and then an upgrade again. And those tests passed here.

If you really want to look further, you can read my docs on it :) https://github.com/deep-learning-indaba/Baobab/blob/develop/api/README.md#merging-migrations

**Comment by KaleabTessera** [comment]
Closing for now.

**Pull request merged**